packages/camera/lib/camera.dart

```diff
@@ -218,7 +240,7 @@ class CameraController extends ValueNotifier<CameraValue> {
   /// [path_provider](https://pub.dartlang.org/packages/path_provider).
   ///
   /// Throws a [CameraException] if the capture fails.
-  Future<Null> capture(String path) async {
+  Future<Null> takePicture(String path) async {
```

Comment:
We will have to make a list of breaking changes in the changelog. This is one. I agree that `capture` is no longer a good name.

Reply:
I agree for the *initialize* method, I'll replace *openCamera* with *initialize*.